content/release-10-18-0/announcements-10-18-0.md

```diff
@@ -19,42 +19,12 @@ Find more details in the document [Continuous Deployment for Cumulocity IoT](/fi
 
 ### REST API changes
 
-#### Planned
-
-##### Breaking change in the Inventory API -- change of the return code for the deletion of managed objects
-
-As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.19+, if the deletion of a managed object and its dependencies doesn´t finish immediately, the platform will return a 202 (Accepted) HTTP status code instead of 204 (No content).
-The return code will still be 204 if the control can be returned immediately.
-This change is required to make the API consistent with the HTTP protocol semantics (asynchronous request).
-
-##### Breaking change in the Inventory API -- change of the default value for the `withChildren` parameter
-
-As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of a future release, the default value for the `withChildren` parameter of the `GET /inventory/managedObjects` endpoint will be changed from `true` to `false`.
-As a consequence, if a request is supposed to return child assets, devices and additions, a parameter `?withChildren=true` has to be provided in the query.
-This change is necessary to improve the Inventory API performance.
-
-##### Breaking change in all REST APIs
-
-As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), as of release 10.19+,
-the `totalPages` statistics value by default is no longer returned for REST API requests without search criteria.
-A `withTotalPages=true` parameter has to be explicitly provided to the requests when `totalPages` is expected.
-
-For example:
-`GET /user/users?pageSize=1000` does not return `totalPages` statistics.
-`GET /user/users?withTotalPages=true` does return `totalPages` statistics.
-
-The change is enforced by performance reasons.
-
-##### Change in the Application API (REST API and Java SDK)
-
-In a future release, we will remove the `resourcesUrl` field from the Application API (both REST API and Java SDK). The `resourcesUrl` is a legacy field, and the functionality behind it was removed.
-This change will not affect any user in a negative way, nor break an existing functionality.
 
 #### Implemented
 
 ##### Breaking change in the Alarms, Events, Measurements APIs - required parameters will be introduced
 
-As announced earlier, most recently with [release 10.17](/release-10-17-0/announcements-10-17-0), at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
+As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.
```

Comment:
suggestion: "As announced with [release 10.17](..."
avoids a comma and feels more flowy

Reply:
It has not only been announced with 10.17 but also earlier, which is why I used this phrasing. I use "with 10.17" if it has been announced there for the first time. Anyway, I will no longer use this duplicate announcements in the future.